### PR TITLE
(DOC) clean and clarify setup comments

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

File: tests/scripts/setup [hunk 1]
```diff
@@ -25,7 +25,10 @@ def create_node(opts)
   wait_for_container(opts['name'])
 
   #There doesn't seem to be a way to use existing puppet tooling to generate
-  #a CSR from the agent side outside the context of a puppet run.
+  #a CSR from the agent side outside the context of a puppet run. Instead,
+  #The following commands complete this process from the master and copy the
```
Comment (greglarkin) [inline tests/scripts/setup]: Small nit - lowercase "The"

File: tests/scripts/setup [hunk 2]
```diff
@@ -46,16 +49,19 @@ def run_puppet_on_nodes
...
 
   manifests_dir = '/etc/puppetlabs/code/environments/production/manifests/'
 
   begin
 
     #Using the --tags puppet_enterprise::profile::agent argument ensures that
-    #only the resources in the PE agent profile are run, any compilation errors
-    #still abort the run. For clean catalog compilation, substitute empty site.pp
-    #file, then revert to the original after the runs ar complete.
+    #only the resources in the PE agent profile are run, but compilation errors
```
Comment (greglarkin) [inline tests/scripts/setup]: Small nit - uppercase "only"

Comment (kjhenner) [reply tests/scripts/setup]: @greglarkin This one is correct as is. It's part of a sentence that begins before the deleted lines.

Comment (greglarkin) [reply tests/scripts/setup]: Woops, sorry ;)

**Pull request merged**